src/views/Dashboard.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <v-row>
+    <v-col cols="3">
+
+    </v-col>
+    <v-col cols="6">
```

Comment:
Dashboard is not responsive. It does not look good in mobile. Please fix this. 

There are two ways in which you can go about this. 

1. Just add `xs="12"` and `md="6"` to `v-col`. Similarly for left and right panes. So that in mobile all panes appear below each other.

1. Or Try to make left and right pane collapse to side in mobile view. There should be hamburger icon or something similar on top which should be used to make the sidebars visible.

If you think 2nd implementation is difficult, go ahead with first one.

Reply:
2nd one was the thought. I wanted to make sure if you're okay with this generic list renderer. This commit is just to give you a glance. Will have the required changes by EOD.

Reply:
Oh..Ok. The generic list looks very good. 

Let us take a use-case.

For example, we are rendering list of questions, Assume that each question will have `title`, `category`, `difficulty_level`, `list_of_tags`. 

Now tell me what would you pass to this generic component and in what structure do you need to pass.

Reply:
The component I wrote expects a child component and props that are to be passed.

Reply:
What I wanted to ask was - is there some processing that needs to be done from the response that we get from api. Or we can directly pass the api response to the component and it will take care of rest.

For example, we might get a list of question objects as an api response. Now we have to preprocess that list( everytime we want to use)  to fit the structure that you are using.

So is it possible to remove this extra step?

Reply:
Yes this is possible.